utils/frame/benchmarking-cli/src/lib.rs

```diff
@@ -80,6 +84,18 @@ pub struct BenchmarkCmd {
 	#[structopt(long)]
 	pub extra: bool,
 
+	/// Output files using spaces instead of tabs.
+	#[structopt(long)]
+	pub spaces: bool,
```

Comment:
why?

Reply:
liuchengxu directly asked me in this PR. I see no harm..?